xdsl/traits.py

```diff
@@ -110,3 +112,27 @@ def verify(self, op: Operation) -> None:
                     # too; in which case it will check itself.
                     if not child_op.has_trait(IsolatedFromAbove):
                         regions += child_op.regions
+
+
+class SymbolOpInterface(OpTrait):
+    @staticmethod
+    def get_sym_attr_name(op: Operation) -> StringAttr:
+        """
+        Returns the symbol of the operation
+        """
+        return cast(StringAttr, op.attributes["sym_name"])
+
+    def verify(self, op: Operation) -> None:
+        # import builtin here to avoid circular import
+        from xdsl.dialects.builtin import StringAttr
+
+        if "sym_name" not in op.attributes:
+            raise VerifyException(
+                f"Operation {op.name} must have sym_name to conform to {SymbolOpInterface.__name__}"
```

Comment:
More than 88 characters here

Reply:
Also, "must have sym_name attribute", to make that more explicit.

Reply:
I don't think it's 88 characters that we have in black. 
- [ ] look into splitting the line
- [ ] reword comment